drupalcat/config/install/core.base_field_override.node.drupal_cat_project.promote.yml

```diff
@@ -0,0 +1,22 @@
+uuid: 1c5bd5f0-fcbc-4721-ad01-1aed89e32022
```

Comment:
No deberíamos usar uuids
Punto 3:
http://nuvole.org/blog/2013/sep/03/configuration-management-and-features-look-drupal-8